pkg/selinux/setroubleshoot-client.js

```diff
@@ -107,69 +107,56 @@ client.init = function(capabilitiesChangedCallback) {
       lastSeen: when the alert was seen for the last time, timestamp in ms
       level: "green", "yellow" or "red"
     */
-    client.getAlert = function(localId) {
-        return new Promise((resolve, reject) => {
-            client.proxy.call("get_alert", [localId])
-                    .then(result => {
-                        const details = {
-                            localId: result[0],
-                            summary: result[1],
-                            reportCount: result[2],
-                            auditEvent: result[3],
-                            pluginAnalysis: result[4],
-                            firstSeen: result[5] / 1000,
-                            lastSeen: result[6] / 1000,
-                            level: result[7],
-                        };
-                        // cleanup analysis
-                        details.pluginAnalysis = details.pluginAnalysis.map(itm => ({
-                            ifText: itm[0],
-                            thenText: itm[1],
-                            doText: itm[2],
-                            analysisId: itm[3],
-                            fixable: itm[4],
-                            reportBug: itm[5],
-                        }));
-                        resolve(details);
-                    })
-                    .catch(ex => {
-                        console.warn("Unable to get alert for id " + localId);
-                        console.warn(ex);
-                        reject(new Error(cockpit.format(_("Unable to get alert: $0"), localId)));
-                    });
-        });
-    };
+    client.getAlert = localId => client.proxy.call("get_alert", [localId])
+            .then(result => {
+                const details = {
+                    localId: result[0],
+                    summary: result[1],
+                    reportCount: result[2],
+                    auditEvent: result[3],
+                    pluginAnalysis: result[4],
+                    firstSeen: result[5] / 1000,
+                    lastSeen: result[6] / 1000,
+                    level: result[7],
+                };
+                // cleanup analysis
+                details.pluginAnalysis = details.pluginAnalysis.map(itm => ({
+                    ifText: itm[0],
+                    thenText: itm[1],
+                    doText: itm[2],
+                    analysisId: itm[3],
+                    fixable: itm[4],
+                    reportBug: itm[5],
+                }));
+                return details;
+            })
+            .catch(ex => {
+                console.warn("Unable to get alert for id", localId, ":", JSON.stringify(ex));
+                return Promise.reject(new Error(`Unable to get alert for id ${localId}: ${ex.toString()}`));
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18670-20230419-100003-a9ad7ebb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)